src/components/Directory.component.tsx

```diff
@@ -0,0 +1,21 @@
+import { Card, CardContent, Typography } from '@mui/material'
+import React from 'react'
+import { FileTypes, Folder } from '../model/Folder'
+import FolderIcon from '@mui/icons-material/Folder';
+import DescriptionIcon from '@mui/icons-material/Description';
+
+const Directory = (props: Folder) => {
+    return (
+        <Card variant="outlined">
+            <CardContent className="folderItem">
+                { props.type === FileTypes.Folder ? <FolderIcon /> : <DescriptionIcon />  }
+                
+                <Typography sx={{ fontSize: 14, marginBottom: 0 }} color="text.secondary" gutterBottom>
```

Comment:
Instead of adding inline-style objects, use makeStyles. Refer https://mui.com/system/styles/basics/#hook-api

Reply:
I referred to the documentation.
makeStyles is now deprecated in v5.
sx is the recommended solution as it is assigning styles to css class and not as inline styles.

Reply:
Cool, just look into v4 docs for how to use as we have mui-v4 in some projects. It will be handy.